docs/sb_newBeamtime.rst

```diff
@@ -25,6 +21,7 @@ Setup
 #. **New user will show up tomorrow or today:**
 
   On XPD computer,
+    #. In a new terminal run ``source activate 17Q3-dev`` then ``bluesky-0MQ-proxy 5577 5578`` if this command fails you already had one running and are fine, and if it doesn't fail then you didn't have one running and now you do and are thus also fine.
     #. type ``xpdui`` to start the ``(collection)`` environment and get the ``In[1]:`` prompt.
```

Comment:
same as above,

would it be good to turn inline code highlighting into sphinx code block so that it's easier to follow?

I think that can help a lot.